fluent-react/src/localization.ts

```diff
@@ -73,6 +79,195 @@ export class ReactLocalization {
     return fallback || id;
   }
 
+  getFragment(
+    id: string,
+    args?: {
+      vars?: Record<string, FluentVariable>,
+      elems?: Record<string, ReactElement>,
+    },
+    fallback?: string
+  ): ReactFragment {
+    return this.getElement(createElement(Fragment, null, fallback || id), id, args);
+  }
+
+  getElement(
+    component: ReactNode | Array<ReactNode>,
+    id: string,
+    args?: {
+      vars?: Record<string, FluentVariable>,
+      elems?: Record<string, ReactElement>,
+      attrs?: Record<string, boolean>;
+    },
```

Comment:
Oh heh, good eye. I'd actually argue that renaming `getString`s second argument to `vars` would be more appropriate, since the Fluent docs also [call them variables](https://projectfluent.org/fluent/guide/variables.html), but I can understand that changing the name of an existing API might not be desirable.

I could name it something like `props`? Because they're basically analogous to `<Localized>`'s props, although it could also be confusing, given that they're not React props. Or how about `params`?

Reply:
I'm ok with changing the internal name of the `getSrting()` argument to `vars`, because it only impacts the documentation.

`props` and `params` aren't really better, and both come with their own sets of baggage.